content/examples/docker/basic.docker-compose.yml.md

```diff
@@ -1,4 +1,4 @@
-```yaml
+```yaml title="docker-compose.yaml"
 version: "3"
```

Comment:
I believe `version` is deprecated (https://forums.docker.com/t/do-you-need-to-put-the-version-at-the-top-of-docker-compose-yml-file/135863).

```suggestion
```